src/partials/benefits/benefits.html

```diff
@@ -1 +1,71 @@
-<section class="benefits">BENEFITS</section>
+<section class="benefits">
+  <div class="container">
+    <div class="benefits-wrapper">
+      <h2 class="benefits-section-title">BENEFITS OF WORKING WITH ME</h2>
+      <ul class="benefits-list">
+        <li class="benefits-items-list">
+          <div class="icon-wrapper">
+            <svg class="benefits-icon" width="24" height="24">
+              <title>Expertise</title>As a highly experienced developer, I have deep knowledge of
```

Comment:
text "As a highly experienced developer, I have deep knowl......" after <title> shouldd be in teg <desc>...</desc> in all svg